doc/usecases/plugins/spec/create_specification.md

```diff
@@ -0,0 +1,55 @@
+# Use Case: Writing specification for an application
+
+## Summary
+
+- **Scope:** `spec`
+- **Level:** Developer Goal
```

Comment:
This should be a higher level, as summary of other specification related use cases.

Reply:
Yes, you are right!